src/cli/llm/prompts/infer_arg_name.md

```diff
@@ -0,0 +1,10 @@
+Given the Operation Definition of GraphQL, suggest {{count}} meaningful names for the argument names.
+The name should be concise and preferably a single word.
```

Comment:
We should instruct the LLM to not use any ignored values that we pass in the input.